library/lib/stylesheet.js

```diff
@@ -8,17 +8,17 @@ import hotCssReplacementClient from './hot-css-replacement-client?transpiled-jav
 
 const isWatch = process.env.WATCH
 
-export default [
-  <link href={`/${__webpack_css_chunk_hash__}.css`} rel='stylesheet' type='text/css' key='hotCssReplacementClient_0' />,
-  isWatch && createHotReload(__webpack_websocket_port__, __webpack_css_chunk_hash__, __webpack_chunkname__)
-].filter(Boolean)
+export default __webpack_css_chunk_hashes__
+  .map((cssHash, index) => <link href={`${__webpack_public_path__}${cssHash}.css`} rel='stylesheet' type='text/css' key={'stylesheet_' + index} />)
```

Comment:
The `cssHash` would be a better key

> The best way to pick a key is to use a string that uniquely identifies a list item among its siblings.

https://reactjs.org/docs/lists-and-keys.html